platform/broadcom/sonic-platform-modules-dell/s6000/scripts/s6000_platform.sh

```diff
@@ -76,7 +76,7 @@ switch_board_qsfp_lpmode() {
 
 install_python_api_package() {
     device="/usr/share/sonic/device"
-    platform=$(/usr/local/bin/sonic-cfggen -H -v DEVICE_METADATA.localhost.platform)
+    platform=${PLATFORM:-`/usr/local/bin/sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
```

Comment:
Why replace the `$(...)` syntax with backticks?

Reply:
same as above